ci-operator/config/openshift/loki/openshift-loki-main.yaml

```diff
@@ -90,6 +88,9 @@ tests:
       cli: latest
       commands: |
         oc -n openshift-operators-redhat rollout status deployment loki-operator-controller-manager
+      dependencies:
+      - env: OO_BUNDLE
+        name: loki-operator-bundle
```

Comment:
If I'm reading the docs correctly, then the dependency declaration should be one level up where it originally was, essentially just replacing `OO_INDEX` with `OO_BUNDLE` and using the bundle name directly:

```
dependencies:
  OO_BUNDLE: loki-operator-bundle
```

Similar for the other files.

Reply:
Unfortunately it looks that the docs are incomplete. The makefile target `make update` gives me an error like the following if I leave the `OO_BUNDLE` definition only at the top:
```
time="2023-02-13T13:15:27Z" level=error error="failed to validate configuration /ci-operator/config/openshift/loki/openshift-loki-release-5.6.yaml: Failed resolve MultiStageTestConfiguration: test/test-operator: workflow/optional-operators-ci-generic-claim: dependency \"OO_BUNDLE\" is overridden in [test/test-operator] but not declared in any step"
```